web-calculator.js

```diff
@@ -0,0 +1,12 @@
+var a,b,result;
+function setValues()
+{
+    a = Number(document.getElementById('a').value);
+    b = Number(document.getElementById('b').value);
+}
+function sum()
+{
+    setValues();
```

Comment:
Good job with the fundamentals here. As you'll see as we progress you'd never split these two things up like this as you wouldn't want variables outside of functions. But we'll get there.